### PR TITLE
Set one memory hotplug case to x86 only

### PR DESCRIPTION
**Pull request by chloerh**
According to the feature itself

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by jinqi7** [comment]
Seems the expect result in rhel9 is changed. So, can you  set it only tested in rhel8 ?

**Review by jinqi7 — COMMENTED** [review]
It doesn't fit for rhel9 also.

**Comment by chloerh** [comment]
> It's need to be add to skiplist for rhel9.

@jinqi7 I'd prefer to add it to skiplist, what do you think?

File: libvirt/tests/cfg/memory/memory_hotplug.cfg [hunk 1]
```diff
@@ -3,6 +3,7 @@
     start_vm = no
     variants case:
         - report_failure:
+            only x86_64
```
Comment (dzhengfy) [inline libvirt/tests/cfg/memory/memory_hotplug.cfg:6]: What's the reason for this memory hotplug case will be only supported on x86_64?

**Comment by jinqi7** [comment]
> > It's need to be add to skiplist for rhel9.
> 
> @jinqi7 I'd prefer to add it to skiplist, what do you think?

That's I am suggesting.

**Comment by chloerh** [comment]
Then it's decided, the case will be added to skiplist.
@jinqi7 @dzhengfy Please help to merge this one, thanks.

File: libvirt/tests/cfg/memory/memory_hotplug.cfg (hunk 1 above)
Comment (chloerh) [reply libvirt/tests/cfg/memory/memory_hotplug.cfg:6]: @dzhengfy it's the design of the case, confirmed with Luyao.

**Review by jinqi7 — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Comment by chloerh** [comment]
I was going to added the case to skiplist and then found that @jinqi7 has already done it for me. Thanks @jinqi7 :+1:

**Pull request merged**